src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -262,8 +269,10 @@ const MetaTrader = (() => {
 
     const submit = (e) => {
         e.preventDefault();
+        const $btn_submit = $(e.target);
+        const acc_type    = $btn_submit.attr('acc_type');
 
-        if (show_new_account_popup) {
+        if (acc_type.includes('real') && show_new_account_popup) {
```

Comment:
showing a confirmation popup to a user, who is still able to change currency, only upon a real (not demo) account creation